src/main/java/com/google/api/codegen/transformer/java/JavaSurfaceNamer.java

```diff
@@ -196,4 +198,10 @@ public String getReleaseAnnotation(ReleaseLevel releaseLevel) {
   public String getBatchingDescriptorConstName(Method method) {
     return inittedConstantName(Name.upperCamel(method.getSimpleName()).join("batching_desc"));
   }
+
+  @Override
+  public String getArtifactId(Model model) {
+    List<String> nameTokens = Splitter.on(".").splitToList(model.getServiceConfig().getName());
```

Comment:
I was hoping to use that exact field but unfortunately I don't think there is access to the artman config data from toolkit (these configs files are designed for artman usage)